logistics_planning_purchase/models/purchase_order.py

```diff
@@ -45,6 +45,9 @@ class PurchaseOrder(models.Model):
     ls_transport_type = fields.Selection(
         related="incoterm_id.ls_sale_transport_type",
     )
+    logistics_vehicle_type_id = fields.Many2one(
```

Comment:
As described in current requirements, at this time we won't adapt this field for purchases